src/WpBookingsUiModule.php

```diff
@@ -430,23 +430,14 @@ function () {
         /*
          * Add screen options on bookings management page.
          */
-        add_filter('screen_settings', function ($settings, \WP_Screen $screen) {
+        $this->_attach('screen_settings', function ($event) {
```

Comment:
@romalytvynenko, does this actually work?

Reply:
Yes, it is actually working.

Reply:
Amazing!